### PR TITLE
docstrings: Change single dash to double dash after input variables

### PR DESCRIPTION
**Pull request by gmou3**
This is achieved by using variants of the following regex rule:
```
Find: ([ ]*- [^ *()\[\]^]* )-( )
Replace: $1--$2
```

**For the reviewer**: to view the _diff_ succinctly run
`git diff 10.4.beta7 -U0 | grep '^[+-]' | grep -Ev '^(--- a/|\+\+\+ b/)'`
or see the attached file: [diff.txt](https://github.com/sagemath/sage/files/15447061/diff.txt) (computed from 307cbf9).

Similar command with colors:
`git diff --color 10.4.beta7 -U0 | grep -Ev 'diff|index|@@|---|\+\+\+'`

### CODE REVIEW

File: src/sage/combinat/words/word_infinite_datatypes.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __init__(self, parent, callable, length=None):
         r"""
         INPUT:
 
-        - ``parent`` - a parent
+        - ``parent`` -- a parent
         -  ``callable`` - a callable defined on ``range(stop=length)``
         -  ``length`` - (default: ``None``) nonnegative integer or ``None``
```
Comment (kwankyu) [inline src/sage/combinat/words/word_infinite_datatypes.py:32]: lines 30,31?

File: src/sage/combinat/words/word_infinite_datatypes.py [hunk 2]
```diff
@@ -589,7 +589,7 @@ def __init__(self, parent, iter, length=None):
         r"""
         INPUT:
 
-        - ``parent`` - a parent
+        - ``parent`` -- a parent
```
Comment (kwankyu) [inline src/sage/combinat/words/word_infinite_datatypes.py:592]: The following lines too?

File: src/sage/combinat/words/word_infinite_datatypes.py [hunk 3]
```diff
@@ -922,7 +922,7 @@ def __init__(self, parent, iter, length=None):
...
```
Comment (kwankyu) [inline src/sage/combinat/words/word_infinite_datatypes.py:925]: The same here?

File: src/sage/crypto/lwe.py [hunk 4]
```diff
@@ -253,17 +253,17 @@ def __init__(self, n, q, D, secret_dist='uniform', m=None):
...
+        - ``D`` -- an error distribution such as an instance of
           :class:`DiscreteGaussianDistributionIntegerSampler` or :class:`UniformSampler`
-        - ``secret_dist`` - distribution of the secret (default: 'uniform'); one of
+        - ``secret_dist`` -- distribution of the secret (default: 'uniform'); one of
 
-          - "uniform" - secret follows the uniform distribution in `\Zmod{q}`
-          - "noise" - secret follows the noise distribution
+          - "uniform" -- secret follows the uniform distribution in `\Zmod{q}`
+          - "noise" -- secret follows the noise distribution
           - ``(lb,ub)`` - the secret is chosen uniformly from ``[lb,...,ub]`` including both endpoints
```
Comment (kwankyu) [inline src/sage/crypto/lwe.py:265]: line 264 too?

File: src/sage/matrix/matrix_double_dense.pyx [hunk 5]
```diff
@@ -2220,11 +2220,11 @@ cdef class Matrix_double_dense(Matrix_numpy_dense):
 
         INPUT:
 
-        - ``tol`` - default: ``1e-12`` - the largest value of the
+        - ``tol`` -- default: ``1e-12`` - the largest value of the
           absolute value of the difference between two matrix entries
```
Comment (kwankyu) [inline src/sage/matrix/matrix_double_dense.pyx:2224]: The changes in this file need more touch. For  this instance, 
```
- ``tol`` -- (default: ``1e-12``) the largest value of the ...
```

File: src/sage/matrix/special.py [hunk 6]
```diff
@@ -685,19 +685,19 @@ def diagonal_matrix(arg0=None, arg1=None, arg2=None, sparse=True):
...
       will have an equal number of columns
 
-    - ``ring`` - the ring containing the entries of the
+    - ``ring`` -- the ring containing the entries of the
       diagonal entries.  This may not be specified in
       combination with a NumPy array.
 
-    - ``sparse`` - default: ``True`` - whether or not
+    - ``sparse`` -- default: ``True`` - whether or not
       the result has a sparse implementation.
```
Comment (kwankyu) [inline src/sage/matrix/special.py:700]:
```
- ``sparse`` -- (default: ``True``) whether or not ...
```
more similar changes in this file.

File: src/sage/misc/html.py [hunk 7]
```diff
@@ -269,19 +269,19 @@ def eval(self, x, globals=None, locals=None, mode='display', combine_all=False):
...
 
-        - ``x`` - a Sage object
+        - ``x`` -- a Sage object
 
         -  ``globals`` - a globals dictionary
 
         -  ``locals`` - extra local variables used when
            evaluating Sage code in ``x``.
 
-        - ``mode`` - string (optional, default ``'display'``):
```
Comment (kwankyu) [inline src/sage/misc/html.py:279]: Above lines 274, 277?

File: src/sage/misc/sage_unittest.py [hunk 8]
```diff
@@ -175,7 +175,7 @@ def run(self, category=None, skip=[], catch=True, raise_on_failure=False,
 
          - ``category``         - a category; reserved for future use
          - ``skip``             - a string or list (or iterable) of strings
-         - ``raise_on_failure`` - a boolean (default: False)
+         - ``raise_on_failure`` -- a boolean (default: False)
          - ``catch``            - a boolean (default: True)
```
Comment (kwankyu) [inline src/sage/misc/sage_unittest.py:180]: Other lines?

File: src/sage/modular/modsym/p1list.pyx [hunk 9]
```diff
@@ -49,7 +49,7 @@ cdef int c_p1_normalize_int(int N, int u, int v,
 
     -  ``vv`` - an integer
 
-    - ``ss`` - an integer such that `(ss*uu, ss*vv)` is congruent to
+    - ``ss`` -- an integer such that `(ss*uu, ss*vv)` is congruent to
        `(u,v)` (mod `N`);
```
Comment (kwankyu) [inline src/sage/modular/modsym/p1list.pyx:53]: line 50?

many more below.

File: src/sage/modular/pollack_stevens/dist.pyx [hunk 10]
```diff
@@ -65,7 +65,7 @@ def get_dist_classes(p, prec_cap, base, symk, implementation):
 
     - ``symk``     -- An element of Symk
 
-    - ``implementation`` - string - If not None, override the
+    - ``implementation`` -- string - If not None, override the
       automatic choice of implementation. May be 'long' or 'vector',
```
Comment (kwankyu) [inline src/sage/modular/pollack_stevens/dist.pyx:68]: Treat line 66?

File: src/sage/modules/vector_double_dense.pyx [hunk 11]
```diff
@@ -367,7 +367,7 @@ cdef class Vector_double_dense(Vector_numpy_dense):
 
         INPUT:
 
-        - ``p`` - default: 2 - controls which norm is computed,
+        - ``p`` -- default: 2 - controls which norm is computed,
           allowable values are any real number and positive and
```
Comment (kwankyu) [inline src/sage/modules/vector_double_dense.pyx:371]:
```
- ``p`` -- (default: 2) controls which norm is computed,
```

File: src/sage/rings/integer.pyx [hunk 12]
```diff
@@ -3860,28 +3860,28 @@ cdef class Integer(sage.structure.element.EuclideanDomainElement):
 
         -  ``algorithm`` - string
 
-           - ``'pari'`` - (default) use the PARI library
```
Comment (kwankyu) [inline src/sage/rings/integer.pyx:3863]: line 3861?

File: src/sage/rings/polynomial/laurent_polynomial_mpair.pyx [hunk 13]
```diff
@@ -1734,7 +1734,7 @@ cdef class LaurentPolynomial_mpair(LaurentPolynomial):
 
         INPUT:
 
-        - ``root`` - boolean (default ``False``) - if set to ``True``
+        - ``root`` -- boolean (default ``False``) - if set to ``True``
           then return a pair ``(True, sqrt)`` with ``sqrt`` a square
```
Comment (kwankyu) [inline src/sage/rings/polynomial/laurent_polynomial_mpair.pyx:1738]:
```
- ``root`` -- (default ``False``) if set to ``True``
```

File: src/sage/rings/polynomial/multi_polynomial_ideal.py [hunk 14]
```diff
@@ -1349,12 +1349,12 @@ def _groebner_basis_ginv(self, algorithm="TQ", criteria='CritPartially', divisio
...
         INPUT:
 
-        - ``algorithm`` - "TQ", "TQBlockHigh", "TQBlockLow" or "TQDegree"
-        - ``criteria`` - "Without" (without any criteria)
+        - ``algorithm`` -- "TQ", "TQBlockHigh", "TQBlockLow" or "TQDegree"
+        - ``criteria`` -- "Without" (without any criteria)
                         - "C1", "CritPartially" (partial involutive criteria)
                         - "C1C2C3", "C1C2C3C4" (full involutive criteria)
 
-        - ``division_interface`` - either "Janet" or "JanetLike"
```
Comment (kwankyu) [inline src/sage/rings/polynomial/multi_polynomial_ideal.py:1357]: Together with line 1354,1455, manual editing is needed.

File: src/sage/rings/polynomial/pbori/pbori.pyx [hunk 15]
```diff
@@ -260,10 +260,10 @@ cdef class BooleanPolynomialRing(BooleanPolynomialRing_base):
 
     -  ``n`` - number of variables (an integer > 1)
 
-    - ``names`` - names of ring variables, may be a string or
```
Comment (kwankyu) [inline src/sage/rings/polynomial/pbori/pbori.pyx:263]: line 261?

File: src/sage/rings/polynomial/pbori/pbori.pyx [hunk 16]
```diff
@@ -4817,7 +4817,7 @@ class BooleanPolynomialIdeal(MPolynomialIdeal):
 
         -  ``gens`` - a list of generators
 
-        - ``coerce`` - coerce all elements to the ring ``ring`` (default: ``True``)
```
Comment (kwankyu) [inline src/sage/rings/polynomial/pbori/pbori.pyx:4820]: line 4818?

File: src/sage/rings/polynomial/polynomial_zmod_flint.pyx [hunk 17]
```diff
@@ -176,7 +176,7 @@ cdef class Polynomial_zmod_flint(Polynomial_template):
 
         INPUT:
 
-        - ``x`` - a list of coefficients - the coefficients are assumed to be
+        - ``x`` -- a list of coefficients - the coefficients are assumed to be
           reduced already and the list contains no trailing zeroes.
```
Comment (kwankyu) [inline src/sage/rings/polynomial/polynomial_zmod_flint.pyx:180]: Maybe replace ` - ` with `;`. Not sure about this...

File: src/sage/schemes/cyclic_covers/cycliccover_generic.py [hunk 18]
```diff
@@ -51,9 +51,9 @@ def __init__(self, AA, r, f, names=None):
 
         INPUT:
 
-        - ``A`` - ambient affine space
+        - ``A`` -- ambient affine space
 
-        - ``r`` - degree of the cover
+        - ``r`` -- degree of the cover
```
Comment (kwankyu) [inline src/sage/schemes/cyclic_covers/cycliccover_generic.py:56]: line 58?

File: src/sage/libs/singular/option.pyx [hunk 19]
```diff
@@ -476,26 +476,26 @@ cdef class LibSingularVerboseOptions(LibSingularOptions_abstract):
...
-     - ``usage`` - shows correct usage in error messages.
+     - ``usage`` -- shows correct usage in error messages.
      - ``Imap`` or ``imap`` - shows the mapping of variables with
        the ``fetch`` and ``imap`` commands.
      - ``notWarnSB`` or ``not_warn_sb`` - do not warn if
        a basis is not a standard basis
      - ``contentSB`` or ``content_sb`` - avoids to divide by the
        content of a polynomial in ``std`` and related algorithms.
        Should usually not be used.
-     - ``cancelunit`` - avoids to divide polynomials by non-constant
```
Comment (kwankyu) [inline src/sage/libs/singular/option.pyx:498]: Untreated items above?

File: src/sage/categories/groups.py [hunk 20]
```diff
@@ -224,7 +224,7 @@ def cayley_table(self, names='letters', elements=None):
               * a list - a list of strings, where the length
                 of the list equals the number of elements.
 
-            - ``elements`` - default = ``None``.  A list of
+            - ``elements`` -- default = ``None``.  A list of
               elements of the group, in forms that can be
```
Comment (kwankyu) [inline src/sage/categories/groups.py:228]:
```
- ``elements`` -- (default: ``None``)  a list of
```

File: src/sage/graphs/graph_generators.py [hunk 21]
```diff
@@ -1543,10 +1543,10 @@ def fullerenes(self, order, ipr=False):
 
         INPUT:
 
-        - ``order`` - a positive even integer smaller than or equal to 254.
+        - ``order`` -- a positive even integer smaller than or equal to 254.
           This specifies the number of vertices in the generated fullerenes.
 
-        - ``ipr`` - default: ``False`` - if ``True`` only fullerenes that
+        - ``ipr`` -- default: ``False`` - if ``True`` only fullerenes that
           satisfy the Isolated Pentagon Rule are generated. This means that
```
Comment (kwankyu) [inline src/sage/graphs/graph_generators.py:1550]:
```
- ``ipr`` -- (default: ``False``) if ``True`` only fullerenes that
```

Comment (kwankyu) [reply src/sage/graphs/graph_generators.py:1550]: Many similar changes needed in this file...

**Comment by kwankyu** [comment]
Nice work.

**Comment by gmou3** [comment]
@kwankyu thanks a lot! I used your suggestions to apply more rules and changes.
I also applied the one you designated as manual labor.

There may be more things to improve; anyone who finds something please share.

**A new question emerges for me:** Should it actually be two dashes (en-dash), or three dashes (em-dash)?
In terms of general typography, it seems that the em-dash is the correct answer.

File: src/sage/algebras/letterplace/free_algebra_letterplace.pyx [hunk 22]
```diff
@@ -644,8 +644,8 @@ cdef class FreeAlgebra_letterplace(Algebra):
 
         INPUT:
 
-        ``g`` - a list of elements of this free algebra.
-        ``d`` - an integer.
+        ``g`` -- a list of elements of this free algebra.
+        ``d`` -- an integer.
```
Comment (kwankyu) [inline src/sage/algebras/letterplace/free_algebra_letterplace.pyx:649]: leading `-`?

File: src/sage/algebras/steenrod/steenrod_algebra.py [hunk 23]
```diff
@@ -520,12 +520,12 @@ def __init__(self, p=2, basis='milnor', **kwds):
...
 
-        - ``p`` -- positive prime integer (optional, default 2)
-        - ``basis`` -- string (optional, default = 'milnor')
-        - ``profile`` -- profile function (optional, default ``None``)
-        - ``truncation_type`` -- (optional, default 'auto')
-        - ``precision`` -- (optional, default ``None``)
-        - ``generic`` -- (optional, default 'auto')
+        - ``p`` -- positive prime integer (default: 2)
+        - ``basis`` -- string (default: = 'milnor')
+        - ``profile`` -- profile function (default: ``None``)
```
Comment (kwankyu) [inline src/sage/algebras/steenrod/steenrod_algebra.py:525]:
```
- ``basis`` -- string (default: ``'milnor'``)
```
Similarly below.

File: src/sage/algebras/steenrod/steenrod_algebra.py [hunk 24]
```diff
@@ -2314,7 +2314,7 @@ def P(self, *nums):
 
         INPUT:
 
-        -  ``a, b, c, ...`` - non-negative integers
+        -  ``a, b, c, ...`` -- non-negative integers
```
Comment (kwankyu) [inline src/sage/algebras/steenrod/steenrod_algebra.py:2318]:
```
- ``a``, ``b``, ``c``, ... -- non-negative integers
```

File: src/sage/algebras/steenrod/steenrod_algebra.py [hunk 25]
```diff
@@ -3807,12 +3807,12 @@ def SteenrodAlgebra(p=2, basis='milnor', generic='auto', **kwds):
...
 
-    - ``p`` -- positive prime integer (optional, default = 2)
-    - ``basis`` -- string (optional, default = 'milnor')
-    - ``profile`` -- a profile function in form specified below (optional, default ``None``)
-    - ``truncation_type`` -- 0 or `\infty` or 'auto' (optional, default 'auto')
-    - ``precision`` -- integer or ``None`` (optional, default ``None``)
-    - ``generic`` -- (optional, default 'auto')
+    - ``p`` -- positive prime integer (default: = 2)
+    - ``basis`` -- string (default: = 'milnor')
+    - ``profile`` -- a profile function in form specified below (default: ``None``)
```
Comment (kwankyu) [inline src/sage/algebras/steenrod/steenrod_algebra.py:3812]: The same comment...

File: src/sage/algebras/steenrod/steenrod_algebra.py [hunk 26]
```diff
@@ -4226,7 +4226,7 @@ def Sq(*nums):
 
     INPUT:
 
-    -  ``a, b, c, ...`` - non-negative integers
+    -  ``a, b, c, ...`` -- non-negative integers
```
Comment (kwankyu) [inline src/sage/algebras/steenrod/steenrod_algebra.py:4230]: The same comment

File: src/sage/algebras/steenrod/steenrod_algebra_misc.py [hunk 27]
```diff
@@ -653,12 +653,12 @@ def serre_cartan_mono_to_string(mono, latex=False, generic=False):
       or tuple (e0, n1, e1, n2, ...) when `generic=True`, where each ei is 0 or
       1, and each ni is positive
 
-    - ``latex`` -- boolean (optional, default False), if true, output
+    - ``latex`` -- boolean (default: False), if true, output
       LaTeX string
```
Comment (kwankyu) [inline src/sage/algebras/steenrod/steenrod_algebra_misc.py:658]:
```
``False``
```
But perhaps this change is out of scope of your PR. I am okay if you do not do that here. If you want to do it in the lines you touched, there are many more.

File: src/sage/algebras/steenrod/steenrod_algebra_mult.py [hunk 28]
```diff
@@ -697,9 +697,9 @@ def adem(a, b, c=0, p=2, generic=None):
 
     INPUT:
 
-    - `a`, `b`, `c` (optional) - nonnegative integers, corresponding
+    - `a`, `b`, `c` (optional) -- nonnegative integers, corresponding
       to either `P^a P^b` or (if `c` present) to `P^a \beta^b P^c`
```
Comment (kwankyu) [inline src/sage/algebras/steenrod/steenrod_algebra_mult.py:708]:
```
``a``
```
But out of the scope of your PR.

File: src/sage/calculus/all.py [hunk 29]
```diff
@@ -31,204 +31,4 @@
...
-                vars = [SR.var(name) for name in s.parameters.keys()]
-                result = x(*vars)
-                if isinstance(result, (tuple, list)):
-                    return vector(SR, result).function(*vars)
-                else:
-                    return SR(result).function(*vars)
-    return SR(x)
-
-
 from . import desolvers
```
Comment (kwankyu) [inline src/sage/calculus/all.py]: ???

File: src/sage/coding/databases.py [hunk 30]
```diff
@@ -197,7 +197,7 @@ def self_orthogonal_binary_codes(n, k, b=2, parent=None, BC=None, equal=False,
        ``b=4``, all doubly even codes are generated). Must be an even positive
        integer.
 
-    -  ``parent``- - Used in recursion (default: ``None``)
+    -  ``parent``- -- Used in recursion (default: ``None``)
```
Comment (kwankyu) [inline src/sage/coding/databases.py:201]: needs manual touch.

File: src/sage/combinat/cluster_algebra_quiver/mutation_class.py [hunk 31]
```diff
@@ -197,7 +197,7 @@ def _dg_canonical_form(dg, frozen=None):
 
     - ``dg`` -- a directed graph having edge labels (a, b) with a > 0
 
-    - ``frozen`` -- list (optional, default []) of frozen vertices
+    - ``frozen`` -- list (default: []) of frozen vertices
```
Comment (kwankyu) [inline src/sage/combinat/cluster_algebra_quiver/mutation_class.py:201]:
```
``[]``
```

File: src/sage/combinat/designs/block_design.py [hunk 32]
```diff
@@ -210,7 +210,7 @@ def ProjectiveGeometryDesign(n, d, F, algorithm=None, point_coordinates=True, ch
       ``algorithm="gap"``. If ``True``, the ground set is indexed by coordinates
       in `\GF{q}^{n+1}`.  Otherwise the ground set is indexed by integers.
 
-    - ``check`` -- (optional, default to ``True``) whether to check the output.
+    - ``check`` -- (default: to ``True``) whether to check the output.
```
Comment (kwankyu) [inline src/sage/combinat/designs/block_design.py:214]: remove `to`

File: src/sage/combinat/designs/twographs.py [hunk 33]
```diff
@@ -103,7 +103,7 @@ def is_regular_twograph(self, alpha=False):
 
         INPUT:
 
-        - ``alpha`` -- (optional, default is ``False``) return the value of
+        - ``alpha`` -- (default: is ``False``) return the value of
           ``alpha``, if possible.
```
Comment (kwankyu) [inline src/sage/combinat/designs/twographs.py:107]: remove `is`.

File: src/sage/combinat/subword_complex.py [hunk 34]
```diff
@@ -1964,9 +1964,9 @@ def _greedy_facet(Q, w, side="negative", n=None, pos=0, l=None, elems=[]):
     - ``Q`` -- a word
     - ``w`` -- an element in the Coxeter group
     - ``side`` -- optional, either ``'negative'`` (default) or ``'positive'``
-    - ``n`` -- an integer (optional, defaults to the length of `Q`)
-    - ``pos`` -- an integer (optional, default 0)
-    - ``l`` -- an integer (optional, defaults to the length of `w`)
+    - ``n`` -- an integer (default:s to the length of `Q`)
+    - ``pos`` -- an integer (default: 0)
+    - ``l`` -- an integer (default:s to the length of `w`)
     - ``elems`` -- a list (optional)
```
Comment (kwankyu) [inline src/sage/combinat/subword_complex.py:1972]: neeeds manual touch...

File: src/sage/graphs/digraph_generators.py [hunk 35]
```diff
@@ -666,7 +666,7 @@ def nauty_directg(self, graphs, options="", debug=False):
                      the part number (first is 0) and the second is the number of
                      parts. Splitting is done per input graph independently.
 
-        - ``debug`` (boolean) -- default: ``False`` - if ``True``
+        - ``debug`` (boolean) -- (default: ``False``); if ``True``
           directg standard error and standard output are displayed.
```
Comment (kwankyu) [inline src/sage/graphs/digraph_generators.py:670]: There are many cases like this. It seems to me that the semicolon is redundant.

File: src/sage/interfaces/fricas.py [hunk 36]
```diff
@@ -1705,12 +1705,12 @@ def _sage_expression(fricas_InputForm):
...
                         |       2                    2
-                       \|- 3 %y1  - 2 %y0 %y1 - 3 %y0   - %y1 - %y0
+                       \|- 3 %y1  - 2 %y0 %y1 - 3 %y0   - %y1 -- %y0
             [%y0, %y1, --------------------------------------------,
                                              2
                 +-----------------------------+
                 |       2                    2
-             - \|- 3 %y1  - 2 %y0 %y1 - 3 %y0   - %y1 - %y0
+             - \|- 3 %y1  - 2 %y0 %y1 - 3 %y0   - %y1 -- %y0
              ----------------------------------------------]
```
Comment (kwankyu) [inline src/sage/interfaces/fricas.py]: Revert!

File: src/sage/interfaces/gnuplot.py [hunk 37]
```diff
@@ -64,15 +64,15 @@ def plot(self, cmd, file=None, verbose=True, reset=True):
         INPUT:
 
 
-        -  ``cmd`` - string
```
Comment (kwankyu) [inline src/sage/interfaces/gnuplot.py:67]: redundant blank line above.

File: src/sage/interfaces/kash.py [hunk 38]
```diff
@@ -542,13 +542,13 @@ def eval(self, x, newlines=False, strip=True, **kwds):
         INPUT:
 
 
-        -  ``s`` - string containing Kash code.
```
Comment (kwankyu) [inline src/sage/interfaces/kash.py:545]: redundant blank line above.

File: src/sage/interfaces/magma.py [hunk 39]
```diff
@@ -420,13 +420,13 @@ def _read_in_file_command(self, filename):
         INPUT:
 
 
-        -  ``filename`` - string
+        -  ``filename`` -- string
```
Comment (kwankyu) [inline src/sage/interfaces/magma.py:424]: redundant blank lines above and below.

File: src/sage/interfaces/maple.py [hunk 40]
```diff
@@ -751,7 +751,7 @@ def source(self, s):
         INPUT:
 
 
-        -  ``s`` - a string representing the function whose
+        -  ``s`` -- a string representing the function whose
```
Comment (kwankyu) [inline src/sage/interfaces/maple.py:754]: redundant blank lines.

File: src/sage/lfunctions/lcalc.py [hunk 41]
```diff
@@ -211,18 +211,18 @@ def values_along_line(self, s0, s1, number_samples, L=''):
...
 
-        -  ``L`` - defines `L`-function (default:
+        -  ``L`` -- defines `L`-function (default:
            Riemann zeta function)
 
 
         OUTPUT:
 
 
-        -  ``list`` - list of pairs (s, zeta(s)), where the s
```
Comment (kwankyu) [inline src/sage/lfunctions/lcalc.py:225]: redundant blank line above and below

File: src/sage/lfunctions/sympow.py [hunk 42]
```diff
@@ -247,15 +247,15 @@ def analytic_rank(self, E):
...
 
 
         OUTPUT:
 
 
-        -  ``integer`` - analytic rank
+        -  ``integer`` -- analytic rank
 
-        -  ``string`` - leading coefficient (as string)
+        -  ``string`` -- leading coefficient (as string)
```
Comment (kwankyu) [inline src/sage/lfunctions/sympow.py:258]: Wonder why there are many blank lines...

File: src/sage/matrix/matrix2.pyx [hunk 43]
```diff
@@ -5981,15 +5981,15 @@ cdef class Matrix(Matrix1):
...
+        -  ``self`` -- Matrix
 
-        -  ``v`` - Vector
+        -  ``v`` -- Vector
 
 
         OUTPUT:
 
 
-        -  ``list`` - list of Vectors
```
Comment (kwankyu) [inline src/sage/matrix/matrix2.pyx:5992]: blank lines again.

File: src/sage/matrix/matrix_gfpn_dense.pyx [hunk 44]
```diff
@@ -873,7 +873,7 @@ cdef class Matrix_gfpn_dense(Matrix_dense):
         INPUT:
 
         - `i`: Index of the first row to be extracted
-        - `j` (optional, default -1): -1, or index of the last
+        - `j` (default: -1): -1, or index of the last
           row to be extracted.
```
Comment (kwankyu) [inline src/sage/matrix/matrix_gfpn_dense.pyx]: `--`?

File: src/sage/matrix/matrix_integer_dense.pyx [hunk 45]
```diff
@@ -2131,25 +2131,25 @@ cdef class Matrix_integer_dense(Matrix_dense):
...
-        -  ``max_dets`` - (default: 5); technical parameter -
+        -  ``max_dets`` -- (default: 5); technical parameter -
            max number of determinant to compute when bounding prime divisor of
            self in its saturation.
 
 
         OUTPUT:
 
 
-        -  ``matrix`` - a matrix over ZZ
```
Comment (kwankyu) [inline src/sage/matrix/matrix_integer_dense.pyx:2152]: Perhaps removing these redundant blank lines should be a separate PR.

File: src/sage/misc/sphinxify.py [hunk 46]
```diff
@@ -39,7 +39,7 @@ def sphinxify(docstring, format='html'):
 
     - ``docstring`` -- string -- a ReST-formatted docstring
 
-    - ``format`` -- string (optional, default 'html') -- either 'html' or
+    - ``format`` -- string (default: 'html') -- either 'html' or
       'text'
```
Comment (kwankyu) [inline src/sage/misc/sphinxify.py:43]: `;`?

File: src/sage/rings/polynomial/multi_polynomial_ideal.py [hunk 47]
```diff
@@ -3037,7 +3037,7 @@ def hilbert_series(self, grading=None, algorithm='sage'):
             sage: J.hilbert_series(grading=(10,3))                                      # needs sage.libs.flint
             (t^25 + t^24 + t^23 - t^15 - t^14 - t^13 - t^12 - t^11
              - t^10 - t^9 - t^8 - t^7 - t^6 - t^5 - t^4 - t^3 - t^2
-             - t - 1)/(t^12 + t^11 + t^10 - t^2 - t - 1)
+             - t -- 1)/(t^12 + t^11 + t^10 - t^2 - t - 1)
```
Comment (kwankyu) [inline src/sage/rings/polynomial/multi_polynomial_ideal.py]: revert.

File: src/sage/schemes/elliptic_curves/gal_reps.py [hunk 48]
```diff
@@ -1108,7 +1108,7 @@ def image_classes(self,p,bound=10000):
 
         - a prime ``p``
 
-        - a natural number ``bound`` (optional, default=10000)
+        - a natural number ``bound`` (default:=10000)
```
Comment (kwankyu) [inline src/sage/schemes/elliptic_curves/gal_reps.py:1112]: `: `

File: src/sage/schemes/generic/algebraic_scheme.py [hunk 49]
```diff
@@ -839,10 +839,10 @@ def rational_points(self, **kwds):
 
         kwds:
 
-        - ``bound`` - integer (optional, default=0). The bound for the coordinates for
+        - ``bound`` -- integer (default:=0). The bound for the coordinates for
           subschemes with dimension at least 1.
```
Comment (kwankyu) [inline src/sage/schemes/generic/algebraic_scheme.py:843]: `: ` here and below

File: src/sage/schemes/projective/projective_homset.py [hunk 50]
```diff
@@ -486,7 +486,7 @@ def points(self, B=0):
 
         INPUT:
 
-        - ``B`` -- integer (optional, default=0). The bound for the
+        - ``B`` -- integer (default:=0). The bound for the
           coordinates.
```
Comment (kwankyu) [inline src/sage/schemes/projective/projective_homset.py:490]: `: `?

File: src/sage/schemes/projective/projective_point.py [hunk 51]
```diff
@@ -1063,7 +1063,7 @@ class SchemeMorphism_point_projective_field(SchemeMorphism_point_projective_ring
 
     - ``v`` -- a list or tuple of coordinates in `K`.
 
-    - ``check`` -- boolean (optional, default:``True``). Whether to
+    - ``check`` -- boolean (default:``True``). Whether to
       check the input for consistency.
```
Comment (kwankyu) [inline src/sage/schemes/projective/projective_point.py:1067]: `: ` (blank)

File: src/sage/schemes/toric/morphism.py [hunk 52]
```diff
@@ -790,7 +790,7 @@ class SchemeMorphism_fan_toric_variety(SchemeMorphism, Morphism):
       fans equal the fans of the domain and codomain in the ``parent``
       Hom-set.
 
-    - ``check`` -- boolean (optional, default:``True``). Whether to
+    - ``check`` -- boolean (default:``True``). Whether to
       check the input for consistency.
```
Comment (kwankyu) [inline src/sage/schemes/toric/morphism.py:794]: `: ` (blank)

File: src/sage/calculus/all.py (hunk 29 above)
Comment (gmou3) [reply src/sage/calculus/all.py]: This is incoming from develop.

File: src/sage/interfaces/fricas.py (hunk 36 above)
Comment (gmou3) [reply src/sage/interfaces/fricas.py]: :O

File: src/sage/algebras/steenrod/steenrod_algebra_mult.py (hunk 28 above)
Comment (gmou3) [reply src/sage/algebras/steenrod/steenrod_algebra_mult.py:708]: I'll leave these for another PR.

File: src/sage/algebras/steenrod/steenrod_algebra_misc.py (hunk 27 above)
Comment (gmou3) [reply src/sage/algebras/steenrod/steenrod_algebra_misc.py:658]: Done.

**Comment by gmou3** [comment]
@kwankyu thanks for the review! I might do a separate PR for the blanklines.
(You answered my previous comment by editing it :/)

File: src/sage/rings/polynomial/infinite_polynomial_ring.py [hunk 53]
```diff
@@ -522,8 +522,8 @@ def __init__(self, parent, start):
         """
         INPUT:
 
-        ``parent`` -- a ring.
-        ``start`` -- some dictionary, usually the dictionary of variables of ``parent``.
+        - ``parent`` -- a rin
+        - ``start`` -- some dictionary, usually the dictionary of variables of ``parent``.
```
Comment (kwankyu) [inline src/sage/rings/polynomial/infinite_polynomial_ring.py]: `a ring.`

File: src/sage/rings/polynomial/infinite_polynomial_ring.py [hunk 54]
```diff
@@ -669,9 +669,9 @@ def __init__(self, R, names, order):
         """
         INPUT:
 
-        ``R`` -- base ring.
-        ``names`` -- list of generator names.
-        ``order`` -- string determining the monomial order of the infinite polynomial ring.
+        - ``R`` -- base rin
+        - ``names`` -- list of generator name
```
Comment (kwankyu) [inline src/sage/rings/polynomial/infinite_polynomial_ring.py]: `base ring`

`generator names`

File: src/sage/rings/polynomial/symmetric_ideal.py [hunk 55]
```diff
@@ -184,9 +184,9 @@ def __init__(self, ring, gens, coerce=True):
         """
         INPUT:
 
-        ``ring`` -- an infinite polynomial ring
-        ``gens`` -- generators of this ideal
-        ``coerce`` -- (bool, default ``True``) coerce the given generators into ``ring``
+        - ``ring`` -- an infinite polynomial ri
+        - ``gens`` -- generators of this ide
+        - ``coerce`` -- (bool, default ``True``) coerce the given generators into ``ring``
```
Comment (kwankyu) [inline src/sage/rings/polynomial/symmetric_ideal.py:189]: `ring`

`ideal`

**Comment by kwankyu** [comment]
Otherwise lgtm.

**Comment by kwankyu** [comment]
> (You answered my previous comment by editing it :/)

Ah, sorry. I hate my elevated privilege :-)

**Comment by kwankyu** [comment]
> ... In terms of general typography, it seems that the em-dash is the correct answer.

I don't know. en-dash is good enough and  inow an established convention. Let's stick to it.

**Comment by gmou3** [comment]
Sure. More than enough changes for one PR.

**Review by kwankyu — APPROVED** [review]
Thanks.

**Comment by kwankyu** [comment]
Next hurdle is to manage merge conflicts.
